<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>637</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>ЕГЭ-тренажер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Bold Arch</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Главное меню</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>311</width>
      <height>451</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="adverbButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Наречия</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pronounButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Местоимения</string>
     </property>
    </widget>
    <widget class="QPushButton" name="accentButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Ударения</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
